examples/test_flow_dir/test_flow_dir.c

```diff
@@ -151,32 +160,96 @@ do_stats_display(struct rte_mbuf *pkt) {
         }
 }
 
+/*
+ * This function populates a set of flows to the sdn flow table.
+ * For each new flow a service chain is created and appended.
+ * Each service chain is of max 4 length. This function is not enabled by default.
+ */
+
+static void
+populate_ipv4_flow_table(void) {
+        struct test_add_key {
+                struct onvm_ft_ipv4_5tuple key;
+                uint8_t destinations[ONVM_MAX_CHAIN_LENGTH];
+        };
+
+        struct test_add_key keys[] = {
+                {{IPv4(101, 0, 0, 0), IPv4(100, 10, 0, 1),  0, 1, IPPROTO_TCP}, {1,2,3,4}},
+                {{IPv4(102, 0, 0, 0), IPv4(101, 10, 0, 1),  0, 1, IPPROTO_TCP}, {4,3,2,1}},
+                {{IPv4(103, 0, 0, 0), IPv4(102, 10, 0, 1),  0, 1, IPPROTO_TCP}, {2,1}},
+        };
+
+        uint32_t num_keys = RTE_DIM(keys);
+        for (uint32_t i = 0; i < num_keys; i++) {
+                struct onvm_ft_ipv4_5tuple key;
+                memset(&key, 0, sizeof(struct onvm_ft_ipv4_5tuple));
+                key = keys[i].key;
```

Comment:
Why do we memset to 0 before reassigning?

Reply:
Updated. Thanks!